apps/mobile/src/components/wallet-settings/account-card.tsx

```diff
@@ -7,10 +9,14 @@ export function AccountCard({
   Icon,
   name,
   onPress,
+  onLongPress,
+  onLayout,
 }: {
   Icon: React.FC<IconProps>;
   name: string;
   onPress?(): unknown;
+  onLongPress?(): unknown;
+  onLayout?(e: LayoutChangeEvent): void;
```

Comment:
Not sure why mixing void and unknown. If no return value expected, `void` should be used imo, as done in docs https://www.typescriptlang.org/docs/handbook/2/functions.html